### PR TITLE
Natural language response

### PR DESCRIPTION
**Pull request by wendecoder**
- Made to the metta output to be converted into natural language.
- Added more example queries.

### CODE REVIEW

File: examples/bio_ai/motto/agents/biochatter/metta_prompt.py [hunk 1]
```diff
@@ -418,10 +587,11 @@ def get_metta_prompt(self) -> str:
             Again, from '(translates_to $transcript $protein)' $protien will be retrieved and will be passed to '(go_gene_product $ontology $protein)'.\
             Finally, from '(go_gene_product $ontology $protein)', $ontology will be retrieved and will be passed to '(subontology $ontology <some_subontology_val>)'. At the end value of $ ontology will be returned."
```
Comment (tanksha) [inline examples/bio_ai/motto/agents/biochatter/metta_prompt.py]: There is a typo here, remove the space from the variable name "$ ontology"

File: examples/bio_ai/motto/agents/biochatter/metta_prompt.py [hunk 2]
```diff
@@ -446,5 +618,5 @@ def get_metta_prompt(self) -> str:
             f"Return only query,  no explanation and other texts"
             f"Based on the information given to you above, you will write a pattern matching query on the dataset for the user's question."
         )
-        # print(prompt)
+        print(prompt)
         return prompt
```
Comment (tanksha) [inline examples/bio_ai/motto/agents/biochatter/metta_prompt.py]: add a newline at the end of the file

File: examples/bio_ai/motto/llm_gate.py [hunk 3]
```diff
@@ -231,7 +232,8 @@ def llm(metta: MeTTa, *args):
                 if func["parameters"]["properties"][k]['metta-type'] == 'Atom':
                     args[k] = metta.parse_single(v)
         return [E(fs, to_nested_expr(list(args.values())), msgs_atom)]
-    print(f'this is {agent} type: {type(response.content)}')
+    # print(f'this is {agent} type: {type(response.content)}')
```
Comment (tanksha) [inline examples/bio_ai/motto/llm_gate.py]: Please clean up commented debug messages

**Review by tanksha — COMMENTED** [review]
Thank you @wendecoder, Please fix some minor comments I left and I will merge it.

File: examples/bio_ai/motto/agents/biochatter/metta_prompt.py (hunk 1 above)
Comment (wendecoder) [reply examples/bio_ai/motto/agents/biochatter/metta_prompt.py]: removed.

File: examples/bio_ai/motto/agents/biochatter/metta_prompt.py (hunk 2 above)
Comment (wendecoder) [reply examples/bio_ai/motto/agents/biochatter/metta_prompt.py]: Added.

File: examples/bio_ai/motto/llm_gate.py (hunk 3 above)
Comment (wendecoder) [reply examples/bio_ai/motto/llm_gate.py]: Cleaned up.

**Comment by wendecoder** [comment]
> Thank you @wendecoder, Please fix some minor comments I left and I will merge it.

> Np @tanksha, I've fixed all the comments you left and have pushed the changes.

**Pull request merged**